lib/data/mailbox.ts

```
@@ -135,3 +135,7 @@ export const updateGitHubRepo = async (mailboxId: number, repoOwner: string, rep
     })
     .where(eq(mailboxes.id, mailboxId));
 };
+
+export const getAllMailboxes = async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)